### ISSUE TITLE
Performance on IMMREP25 dataset

### ISSUE THREAD

**Issue by justin-barton** [VarunUllanat/mint#5]
Hi @VarunUllanat ,

As you mention T cell receptor-epitope binding prediction directly in the preprint, it would be great to see how MINT performs on the IMMREP25 competition dataset:
https://www.kaggle.com/competitions/immrep25

I have had a look in the downstream/TCR-Epitope directory and I can see some training scripts but no obvious inference code that could be used. Could you advise? Or better yet, submit predictions yourselves? Note that the competition closes at the end of this week. Happy to help out if you need any reformatting or processing of the test set (https://www.kaggle.com/competitions/immrep25/data?select=test.csv).

Thanks!

**Comment by VarunUllanat** [comment]
Hello @justin-barton, thanks for your interest in MINT! I will check if I have the model checkpoint for this task.
